server/auth/types/jwt/jwt_helper.test.ts

```diff
@@ -200,4 +213,198 @@ describe('test jwt auth library', () => {
 
     expect(headers).toEqual(expectedHeaders);
   });
+}); // re-import JWTAuth to change cookie splitter to a no-op
+
+/* eslint-disable no-shadow, @typescript-eslint/no-var-requires */
+describe('JWT Expiry Tests', () => {
+  const setExtraAuthStorageMock = jest.fn();
+  jest.resetModules();
+  jest.doMock('../../../session/cookie_splitter', () => ({
+    setExtraAuthStorage: setExtraAuthStorageMock,
+  }));
+  const { JwtAuthentication } = require('./jwt_auth');
+
+  const realDateNow = Date.now.bind(global.Date);
+  const dateNowStub = jest.fn(() => 0);
+  global.Date.now = dateNowStub;
+  const coreSetup = jest.fn();
+
+  afterAll(() => {
+    global.Date.now = realDateNow;
```

Comment:
What does resetting the global Date do? If this test suite takes 1 min to run, would this mean that the clock goes back 1 min?